### PR TITLE
FP: Confusion should not use the HRS to deal damage

### PR DESCRIPTION
**Pull request by Spandamn**
A shoutout to @Nixola and Monodyy for finding this bug.

### CODE REVIEW

**Comment by Spandamn** [comment]
@TheImmortal I always forget to tag you in the main post

File: mods/fullpotential/scripts.js [hunk 1]
```diff
@@ -73,6 +73,9 @@ exports.BattleScripts = {
 		if (move.useTargetOffensive) {
 			attackStat = category === 'Physical' ? 'atk' : 'spa';
 		}
+		if (move.id === 'confusion') {
```
Comment (Marty-D) [inline mods/fullpotential/scripts.js]: `confusion` is already a move id. Why not just add the id `confused` to the confusion damage check at the top of this file and forget about overriding statuses.js altogether?

Comment (Spandamn) [inline mods/fullpotential/scripts.js]: Agreed. Wasn't sure what else uses the confusion damage check, so i thought overriding statuses.js would be better.
Will make the change

**Comment by KrisOnline** [comment]
zzzzz it's not called the hrs

**Pull request merged**